src/Directory.Build.props

```diff
@@ -0,0 +1,36 @@
+<Project>
+  <PropertyGroup>
+    <Company>Public Domain</Company>
+    <Copyright>Public Domain</Copyright>
+    <PackageDescription>Jwt.Net, a JWT (JSON Web Token) implementation for .NET</PackageDescription>
+    <Authors>Alexander Batishchev, John Sheehan, Michael Lehenbauer</Authors>
+    <PackageTags>jwt;json</PackageTags>
+    <PackageLicenseExpression>CC0-1.0</PackageLicenseExpression> <!-- creative commons zero 1.0 -->
+    <PackageProjectUrl>https://github.com/jwt-dotnet/jwt</PackageProjectUrl>
+    <PackageReleaseNotes>https://github.com/jwt-dotnet/jwt/releases</PackageReleaseNotes>
+    <RepositoryUrl>https://github.com/jwt-dotnet/jwt</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+
+    <!-- disable sourcelink on mono, to workaround https://github.com/dotnet/sourcelink/issues/155 -->
+    <EnableSourceLink Condition=" '$(OS)' != 'Windows_NT' AND '$(MSBuildRuntimeType)' != 'Core' ">false</EnableSourceLink>
+    <EnableSourceControlManagerQueries>$(EnableSourceLink)</EnableSourceControlManagerQueries>    
+    <!-- Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element) -->
+    <PublishRepositoryUrl>true</PublishRepositoryUrl> 
+    <!-- Embed source files that are not tracked by the source control manager in the PDB -->
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <!-- Include PDB in the built .nupkg -->
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(SourceLinkEnabled)' != 'false'">
```

Comment:
Should we enable SourceLink as a separate PR as well?

Reply:
Probably. I will likely do four pr then. Introduce directory.build.props, introduce source link, introduce nbgv, change to package reference.

Reply:
Please, much much appreciate your participation and contributions!